lib/view_models/follow_service.dart

```diff
@@ -180,6 +188,13 @@ class FollowService extends BaseModel {
   ///[forFollowersList] is to identify whether we want to perform operation on followers list or following list.
   Future<void> performFollowUnfollow(String atsign,
       {bool forFollowersList: false}) async {
+
+    // check for the atsign we are about to follow is valid or not
+    atStatus = await atStatusImpl.get(atsign);
+    if(atStatus.serverLocation == null) {
+      print('Invalid atSign');
+      return;
+    }
```

Comment:
There should be a message shown when atSign is not valid.

Reply:
yes I have now added a snack bar to display the message